integration-tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import nest_asyncio
+
+
+def pytest_configure(config):
+    # Required for playwright to be run from within pytest
```

Comment:
The alternative to this is to use async tests with the playwright async API. That's what we use in the jupyterhub tests.

Reply:
@minrk oooh, this led me down a hole and I ended up opening https://github.com/jupyterhub/binderhub/issues/1894 to not derail this PR.